x-pack/legacy/plugins/siem/public/pages/hosts/details/index.tsx

```diff
@@ -47,9 +46,7 @@ const HostDetailsComponent = React.memo<HostDetailsComponentProps>(
     isInitializing,
     filterQueryExpression,
     from,
-    match: {
-      params: { detailName, tabName },
-    },
+    detailName,
```

Comment:
Note: In these cases we have to remove `match` and can longer pass it down. Instead we have to destructure the match into its constituent pieces of things such as strings so React.memo does not do additional rendering.